library_generation/utils/utilities.sh

```diff
@@ -111,6 +111,10 @@ get_grpc_version() {
 get_protoc_version() {
   local gapic_generator_version=$1
   local protoc_version
+  if [[ -n "${DOCKER_PROTOC_VERSION}" ]]; then
+    >&2 echo "Using protoc version baked into the container: ${DOCKER_PROTOC_VERSION}"
+    echo "${DOCKER_PROTOC_VERSION}"
+  fi
```

Comment:
I guess we can return this function early?

Reply:
This became obsolete as we moved this if statement outside of `download_protoc`